adaptation.rb

```diff
@@ -86,8 +83,22 @@ def usage(s)
                 t.value = v
                 num_set += 1
             end
+            return
```

Comment:
I'm getting this error here:
```
E: LocalJumpError: ../../code/dfhack/scripts/adaptation.rb:86: unexpected return
```
Are you trying to avoid checking subsequent entries in `misc_traits`? If so, you might want `break` instead.

I see that changing it to `break` results in duplicate output lines, so I'm guessing you're trying to skip the section below. If this `return` works on your end, I'm guessing the issue is that it doesn't work in Ruby 1.8, which is what we use on Linux. Upgrading Ruby to a consistent version on all platforms is a longstanding issue we should address, but I think a workaround in this case could be to set a flag here and check it before printing below.

Reply:
Fixed, I used a flag. Also tested on Linux and everything seems to be working now. Sorry about that.

Reply:
Looks good, thanks!